ansible/roles/test/files/helpers/arp_responder.py

```diff
@@ -106,7 +110,8 @@ def action(self, interface):
         return
         
     def extract_arp_info(self, data):
-        return data[6:12], data[28:32], data[38:42] # remote_mac, remote_ip, request_ip
+        # remote_mac, remote_ip, request_ip, op_type
+        return data[6:12], data[28:32], data[38:42], (ord(data[20]) * 256 + ord(data[21]))
```

Comment:
socket.ntohs(x) ?

Reply:
Sorry Pavel, I tried but didn't find a clean way of doing so. The data array is a byte array. Getting the network order short was the challenge.